src/components/job/JobList.jsx

```diff
@@ -26,40 +26,35 @@ export default function JobList() {
     return data
       ? data.map((job) => {
           return (
-            <li className="flex flex-row justify-center w-full mt-5" key={job.id}>
-              <Link
-                to={job.id}
-                className={`${
+            <li className="flex flex-row w-full mt-5 " key={job.id}>
+              <div
+                className={`card card-compact w-full rounded-xl border-slate-400 border-4 ${
                   job.id && job.id == jobId ? "active" : ""
-                }  w-4/5 rounded-xl border-slate-400 border-4`}
+                }`}
               >
-                <div className="card card-compact w-full">
-                  <div
-                    className={`absolute right-0 translate-y-1/2 bottom-1/2 text-xl ${
-                      auth.user.role === ROLES.CLIENT ? "" : "hidden"
-                    }`}
-                  >
-                    <StarBorderIcon style={{ fontSize: "5vh" }} />
-                  </div>
-                  <div className="card-body">
+                <Link to={job.id} className="w-full flex flex-row justify-between">
+                  <div className="card-body basis-9/12 flex-none">
                     <div className="card-title text-xl">{job.title}</div>
-                    <div className="flex felx-row w-11/12">
-                      <p className="w-1/2 truncate">{job.company.name}</p>
-                      <p className="w-1/2 truncate">{`${job.location}`}</p>
-                    </div>
-                    <div className="flex felx-row w-11/12">
-                      <p className="w-1/2 truncate">
-                        {job.benefits.hasMedical
-                          ? "Medical Benefits"
-                          : "No Medical Benefits"}
-                      </p>
-                      <p className="w-1/2 truncate">
-                        Minimum Pay: {job.wage.min} {job.wage.type}
-                      </p>
+                    <div className="flex flex-row flex-none">
+                      <div className="flex flex-row flex-wrap flex-auto">
+                        <p className="truncate w-1/2">{job.company.name}</p>
+                        <p className="truncate w-1/2">{`${job.location}`}</p>
+                        <p className="truncate w-1/2">
+                          {job.benefits.hasMedical
+                            ? "Medical Benefits"
+                            : "No Medical Benefits"}
+                        </p>
+                        <p className="truncate w-1/2">
+                          Minimum Pay: {job.wage.min} {job.wage.type}
+                        </p>
+                      </div>
                     </div>
                   </div>
```

Comment:
@kendebacker some changes here. could you take a look at the demo page to see the changes in display?

Reply:
Ah yes the flex-none is nice to keep the listing size constant!